src/parser/jobs/mnk/modules/PerfectBalance.tsx

```diff
@@ -0,0 +1,45 @@
+import {t, Trans} from '@lingui/macro'
+import React from 'react'
+
+import {ActionLink} from 'components/ui/DbLink'
+import ACTIONS from 'data/ACTIONS'
+import {Action} from 'data/ACTIONS/ACTIONS'
+import STATUSES from 'data/STATUSES'
+import {Status} from 'data/STATUSES/STATUSES'
+import {BuffWindowModule} from 'parser/core/modules/BuffWindow'
+import {SEVERITY} from 'parser/core/modules/Suggestions'
+
+export default class PerfectBalance extends BuffWindowModule {
+	static handle = 'pb'
+	static title = t('mnk.pb.title')`Perfect Balance Windows`
+
+	buffAction: Action = ACTIONS.PERFECT_BALANCE
+	buffStatus: Status = STATUSES.PERFECT_BALANCE
+
+	expectedGCDs = {
+		expectedPerWindow: 6,
+		suggestionContent: <Trans id="mnk.pb.suggestions.missedgcd.content">
+			Try to land 6 GCDs during every <ActionLink {...ACTIONS.PERFECT_BALANCE} /> window.  If you cannot do this with full uptime and no clipping, consider adjusting your gearset for more Skill Speed.
+		</Trans>,
+		severityTiers: {
+			1: SEVERITY.MEDIUM,
```

Comment:
Many users (especially in Japan) use a different opener that doesn't rely on PB DK+Boot spam. Users with heavy latency will also struggle on this. I think 1 as minor and 2 as medium works, 3 is def major because wtf.

Reply:
Agreeable.

Reply:
The latency was more of a "use PB in the last 3rd of your GCD if you're aiming for 6" kinda thing. I guess what I'm looking for is "always hit 5, but don't necessarily penalise 5 if the user couldn't absolutely get 6." It might be easier to use the matchHighest instead of default matchLowest in the severity, like how RoF does (did?).